clients/lsp-rust.el

```diff
@@ -1663,94 +1670,95 @@ https://github.com/rust-lang/rust-analyzer/blob/master/docs/dev/lsp-extensions.m
   "Init options for rust-analyzer"
   `(:diagnostics
     ( :enable ,(lsp-json-bool lsp-rust-analyzer-diagnostics-enable)
-      :enableExperimental ,(lsp-json-bool lsp-rust-analyzer-diagnostics-enable-experimental)
-      :disabled ,lsp-rust-analyzer-diagnostics-disabled
-      :warningsAsHint ,lsp-rust-analyzer-diagnostics-warnings-as-hint
-      :warningsAsInfo ,lsp-rust-analyzer-diagnostics-warnings-as-info)
+              :enableExperimental ,(lsp-json-bool lsp-rust-analyzer-diagnostics-enable-experimental)
+              :disabled ,lsp-rust-analyzer-diagnostics-disabled
+              :warningsAsHint ,lsp-rust-analyzer-diagnostics-warnings-as-hint
+              :warningsAsInfo ,lsp-rust-analyzer-diagnostics-warnings-as-info)
     :imports ( :granularity ( :enforce ,(lsp-json-bool lsp-rust-analyzer-import-enforce-granularity)
-                              :group ,lsp-rust-analyzer-import-granularity)
-               :group ,(lsp-json-bool lsp-rust-analyzer-import-group)
-               :merge (:glob ,(lsp-json-bool lsp-rust-analyzer-imports-merge-glob))
-               :prefix ,lsp-rust-analyzer-import-prefix)
+                                       :group ,lsp-rust-analyzer-import-granularity)
+                            :group ,(lsp-json-bool lsp-rust-analyzer-import-group)
+                            :merge (:glob ,(lsp-json-bool lsp-rust-analyzer-imports-merge-glob))
+                            :prefix ,lsp-rust-analyzer-import-prefix)
```

Comment:
What's wrong with these indentations? 🤔

Reply:
must be a problem with my formatter, have reapplied patch with formatting turned off